modules/custom/az_paragraphs/az_paragraphs_text_background/az_paragraphs_text_background.module

```diff
@@ -0,0 +1,44 @@
+<?php
+
+/**
+ * @file
+ * Contains az_paragraphs_text_background.module.
+ */
+
+use \Drupal\paragraphs\Entity\Paragraph;
+
+/**
+ * Implements hook_theme_suggestions_HOOK_alter().
+ */
+function az_paragraphs_text_background_theme_suggestions_paragraph_alter(array &$suggestions, array $variables) {
+
+  if (array_key_exists('theme_hook_original', $variables) &&
+    $variables['theme_hook_original'] == 'paragraph') {
+
+    if (array_key_exists('elements', $variables) &&
+      array_key_exists('#paragraph', $variables['elements']) &&
+      $variables['elements']['#paragraph'] instanceof \Drupal\paragraphs\Entity\Paragraph) {
```

Comment:
I think this makes the use statement at the top irrelevant
```
      $variables['elements']['#paragraph'] instanceof \Drupal\paragraphs\Entity\Paragraph) {
```
Instead I think we can do this
```
      $variables['elements']['#paragraph'] instanceof Paragraph) {
```